### PR TITLE
chore: add some notes on chapter 3 of working the command line

### LINKED ISSUE #16: chore: add some notes on chapter 3 of working the command line
I have some basic notes to add to chapter 3 of working the command line.

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds some notes for chapter 3 of working the command line.

fix #16

### CODE REVIEW

**Comment by KobayashiComputing** [comment]
I have reviewed the proposed addition, and generally I think it's great. Two comments, however:

1. In the text regarding the pipe character, I recommend that the word "damn" be replaced with something like "extremely" or "astonishingly" or something similar. I think that there are many people who are still somewhat offended by "damn," even though most simply overlook it. 
2. In the "less cheatsheet," in the first line, I think that changing "up and down cursor" to "up and down arrow keys." I think this will avoid any confusion as to which keys to use.

File: working-the-command-line/chapter3/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Tools of the terminal trade
+
+## The `|` (pipe) chracter
+
+> The pipe is a magic and integral part of the command line. It’s super powerful and damn useful. What it does is take the output from the first command (on the left side) and “pipe” it to the second command (on the right side).
```
Comment (KobayashiComputing) [inline working-the-command-line/chapter3/index.md:5]: Replace the word "damn" with something less potentially objectionable, such as "extremely" or "extraordinarily."

File: working-the-command-line/chapter3/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+81.0.4044.129
+181.214.218.69
+41.0.2224.3
+```
+
+## less
+
+### less cheatsheet
+
+- move up and down using the up and down cursor
```
Comment (KobayashiComputing) [inline working-the-command-line/chapter3/index.md:66]:
```suggestion
- move up and down using the up and down arrow keys
```

**Review by KobayashiComputing — CHANGES_REQUESTED** [review]

File: working-the-command-line/chapter3/index.md (hunk 1 above)
Comment (schalkneethling) [reply working-the-command-line/chapter3/index.md:5]: Ah, yes, good point. That is a direct quote from the book, but I reckon we can clean it up. Thank you for the thoughtful suggestion.

Comment (schalkneethling) [inline working-the-command-line/chapter3/index.md:5]:
```suggestion
> The pipe is a magic and integral part of the command line. It’s super powerful and extremely useful. What it does is take the output from the first command (on the left side) and “pipe” it to the second command (on the right side).
```

**Comment by schalkneethling** [comment]
Thank you for your thorough review, @KobayashiComputing. This one is ready for a re-review. Thank you!

**Review by KobayashiComputing — APPROVED** [review]
Looks good.

**Pull request merged**